pydmd/lando.py

```diff
@@ -0,0 +1,1089 @@
+        """
+        if self._eigenvalues is None:
+            msg = "You need to call fit() and analyze_fixed_point()."
+            raise ValueError(msg)
+        return self._eigenvalues
+
+    @property
+    def eigenvectors(self):
+        """
+        Get the eigenvectors of the reduced linear operator evaluated at a
+        fixed point. The reduced operator is referred to as L_hat in the
+        original reference.
+
+        :return: the eigenvectors of the reduced linear operator.
+        :rtype: numpy.ndarray
+        """
+        if self._eigenvectors is None:
+            msg = "You need to call fit() and analyze_fixed_point()."
+            raise ValueError(msg)
+        return self._eigenvectors
+
+    @property
+    def modes(self):
+        """
+        Get the eigenvectors of the full linear operator evaluated at a
+        fixed point. The full operator is referred to as L in the original
+        reference.
+
+        :return: the eigenvectors of the full linear operator.
+        :rtype: numpy.ndarray
+        """
+        if self._modes is None:
+            msg = "You need to call fit() and analyze_fixed_point()."
+            raise ValueError(msg)
+        return self._modes
+
+    @property
+    def as_numpy_array(self):
+        """
+        Get the reduced linear operator A_tilde.
+        Referred to as L_hat in the original reference.
+
+        :return: the reduced linear operator.
+        :rtype: numpy.ndarray
+        """
+        if self._Atilde is None:
+            msg = "You need to call fit() and analyze_fixed_point()."
+            raise ValueError(msg)
+        return self._Atilde
+
+    @property
+    def A(self):
+        """
+        Get the full linear operator A.
+        Referred to as L in the original reference.
+
+        :return: the full linear operator.
+        :rtype: numpy.ndarray
+        """
+        if self._A is None:
+            msg = (
+                "Full linear operator currently not computed. You may need to "
+                "call fit() and analyze_fixed_point(). If already done so, "
+                "call analyze_fixed_point() using the flag compute_A=True."
+            )
+            warnings.warn(msg)
+        return self._A
+
+    def compute_operator(self, X, Y, kernel_function):
+        """
+        Compute the dictionary-based kernel model weights.
+
+        :param X: the input snapshots x, rescaled.
+        :type X: numpy.ndarray
+        :param Y: input snapshots y such that F(x) = y.
+        :type Y: numpy.ndarray
+        :param kernel_function: kernel function to apply.
+        :type kernel_function: function
+        """
+        # Determine the order in which to parse the data snapshots.
+        parsing_inds = np.arange(X.shape[1])
+        if self._permute:
+            np.random.shuffle(parsing_inds)
+
+        # Initialize the Cholesky factorization routine.
+        ind_0 = parsing_inds[0]
+        x_0 = np.expand_dims(X[:, ind_0], axis=-1)
```

Comment:
I think it's more clear this way:
```suggestion
        x_0 = X[:, ind_0][..., None]
```

Reply:
Done! They should all be changed now.